charts/web3signer/values.yaml

```diff
@@ -21,22 +21,29 @@ extraArgs: []
 # -- Command replacement for the web3signer container
 customCommand: [] # Only change this if you need to change the default command
 
-# -- Set a network name such as mainnet, goerli, sepolia
-network: "prater"
-
 individualKeys:
   enabled: true
 
 # -- Config file
 # @default -- See `values.yaml`
 config: |
   data-path: "/data"
+  eth2.network: prater
   {{- if .Values.individualKeys.enabled }}
   key-store-path: "/data/files"
+  {{- else }}
+  keystores-path: "/data/keys"
+  keystores-password-file: "/data/passwords/keystores.pass"
   {{- end }}
   http-listen-port: {{ .Values.httpPort }}
   http-listen-host: 0.0.0.0
   http-host-allowlist: "*"
+  {{- if .Values.slashingprotectiondb.enabled }}
```

Comment:
you need to have an else, where you strictly disable slashing-protection, otherwise the CI gonna fail.

Reply:
makes sense, ok.